cli/tests/integration_tests.rs

```diff
@@ -1837,6 +1837,11 @@ itest!(cjs_imports {
   output: "cjs_imports.ts.out",
 });
 
+itest!(ts_import_from_js {
+  args: "run --quiet --reload ts_import_from_js.js",
+  output: "ts_import_from_js.js.out",
+});
```

Comment:
This test already passes for me with v1.0.3 ...

Reply:
That's strange; because it fails for me. Make sure to run with `--reload` flag - otherwise compiler might use already compiler file.

Reply:
Changed the test case - it fails on current master and `1.0.3`

Reply:
I confirmed this new test fails on 1.0.3 👍